src/ember-resource.js

```diff
@@ -7,6 +7,14 @@
       getPath = Ember.Resource.getPath,
       set = Ember.set;
 
+  // Determine if we need to be back compatible with Ember < 1.12
+  var requiresEmberComputedPropertyFunction = false;
+  try {
+    Ember.computed({get: function() {}, set: function(key, value) {}});
+  } catch(ex) {
+    requiresEmberComputedPropertyFunction = true;
+  }
+
```

Comment:
Ember throws an error (in production too) if you try to setup a CP without providing a function:
```
Em.computed({get: function() {return null}, set: function(key, value) {return value;}})
vendor-dbf8a96eb99188cbacb68338854fa1dd.js:7 Uncaught n {description: undefined, fileName: undefined, lineNumber: undefined, message: "Computed Property declared without a property function", name: "Error"…}
```